src/libcore/marker.rs

```diff
@@ -363,6 +363,13 @@ pub trait StructuralEq {
 /// [impls]: #implementors
 #[stable(feature = "rust1", since = "1.0.0")]
 #[lang = "copy"]
+// FIXME(matthewjasper) This allows copying a type that doesn't implement
+// `Copy` because of unsatisfied lifetime bounds (copying `A<'_>` when only
+// `A<'static>: Copy` and `A<'_>: Clone`).
```

Comment:
Does this apply only to specialization or to all uses of `Copy`? I'm surprised there is no test that makes sure we fail to copy something when the `Copy` impl is lifetime-dependent.

Reply:
It's only for specializations.